react/react1/week3/homework/todo-list/src/components/TodoListItem.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import "../index.css";
+
+export default function TodoListItem({
+  id,
```

Comment:
Nice with the destructuring in the component definition instead of taking in the props object. It's how I always do it as well :+1: